test/e2e/cypress-ui/README.md

```diff
@@ -0,0 +1,29 @@
+# IDP Cypress Tests
+
+## Running the tests locally
+
+  > Pre-requisites:
+  >  - nodeJS
+  >  - OpenShift CLI (oc)
+  >  - a browser - either chrome or firefox
+
+1. Run `npm install` to install Cypress and it's dependencies on the root directory.
```

Comment:
The `package.json` file is not present in the root directory. I believe this line would need to indicate that `npm install` needs to be run from the `test/e2e/cypress-ui ` directory.